Library/Formula/symlinks.rb

```diff
@@ -0,0 +1,23 @@
+class Symlinks < Formula
+  desc "scan/change symbolic links"
+  homepage "http://www.ibiblio.org/pub/Linux/utils/file/symlinks.lsm"
```

Comment:
Well, seems like I have been both confident and wrong at the same time. `-isysroot` does not automatically affect paths passed via `-I` and judging from the modification date, `/usr/include/malloc` very likely does not exist on non-CLT systems. Sorry! And thanks for questioning me! :bow: